scheds/rust/scx_rusty/src/bpf/main.bpf.c

```diff
@@ -896,6 +933,9 @@ void BPF_STRUCT_OPS(rusty_running, struct task_struct *p)
 		return;
 	}
 
+	s32 cpu = scx_bpf_task_cpu(p);
+	u64 next_vtime_max = p->scx.dsq_vtime - slice_ns;
```

Comment:
I suppose the `- slice_ns` part is to make preemption less aggressive? If so, it might be easier to think about if the adjustment is done when making the preemption decision rather than when remembering the current state.